Wiki/text-notes.md

```diff
@@ -1,107 +1,94 @@
 
-CKEditor supports markdown-like editing experience. It recognizes syntax and automatically converts it to rich text. See it in action:
+```js
+if (1 > 2) {
+    console.log("Error in the matrix");
+}
+```
 
-![](images/autoformat.gif)
+### Headings
 
-Complete documentation for this feature is available in [CKEditor documentation](https://ckeditor.com/docs/ckeditor5/latest/features/autoformat.html).
+Create headings by starting a line with `##` for heading 2, `###` for heading 3, and so on up to heading 6. Note that `#` is reserved for the title.
 
-If the autoformat is not desirable for what you just wrote, you can press `CTRL-Z` which will un-autoformat the text to its original form.
+### Horizontal Line
 
-Note that the use of `#` for Heading1 style is not supported because the editor assumes that is used for the title, start with `##` for Heading2. Explanation [here](https://ckeditor.com/docs/ckeditor5/latest/features/headings.html#heading-levels).
+Insert a horizontal line by starting a line with `---`.
 
-Math support
-------------
+## Markdown & Autoformat
 
-Trilium provides Math support with the help of KaTex:
+CKEditor supports a markdown-like editing experience, recognizing syntax and automatically converting it to rich text.
 
-![](images/math.gif)
+Complete documentation for this feature is available in the [CKEditor documentation](https://ckeditor.com/docs/ckeditor5/latest/features/autoformat.html).
 
-Cut selection to sub-note
--------------------------
+If autoformatting is not desirable, press `CTRL-Z` to revert the text to its original form.
 
-One of the common situations in Trilium is when you're editing a document, and it gets somewhat large, so you start splitting it up into sub-notes - the process is essentially like this:
+Note: The use of `#` for Heading 1 is not supported because it is reserved for the title. Start with `##` for Heading 2. More information is available [here](https://ckeditor.com/docs/ckeditor5/latest/features/headings.html#heading-levels).
 
-*   select the desired piece of text and cut it into clipboard
-*   create new sub-note & give it name
-*   paste the content from clipboard into sub-note
+## Math Support
 
-Trilium provides a way to automate this:
+Trilium provides math support through KaTeX.
 
-![](images/cut-to-subnote.gif)
+## Cutting Selection to Sub-Note
 
-You can notice how heading "Formatting" is automatically detected and new sub-note is named "Formatting".
+When editing a document that becomes too large, you can split it into sub-notes:
 
-It is also possible to assign a keyboard shortcut for this action.
+1. Select the desired text and cut it to the clipboard.
+2. Create a new sub-note and name it.
+3. Paste the content from the clipboard into the sub-note.
 
-Include note
-------------
+Trilium can automate this process. The heading is automatically detected and the new sub-note is named accordingly. You can also assign a keyboard shortcut for this action.
```

Comment:
Trilium can automate this process. Select some text within the note, and in the selection toolbar, click the scissors icon for the "cut & pasted selection to sub-note" action. The heading is automatically detected and the new sub-note is named accordingly. You can also assign a keyboard shortcut for this action.